### PR TITLE
Use JavaScript to Generate Projects' Cards

### PR DESCRIPTION
**Pull request by mohametalmeari**
Use JavaScript array to store all of the information for all projects
Update the main page so that the projects section is created dynamically

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -383,6 +389,57 @@ label {
...
+  justify-content: center;
+  gap: 16px;
+}
+
+.popup_btn button {
+  padding: 8px 16px;
+  display: flex;
+  gap: 8px;
+  align-items: center;
+}
```
Comment (huseyinbiyik) [inline style.css]: Great job with styling 👍  But let's align the buttons to the left like on the Figma design guideline.
Screenshot 2023-04-11 at 10 44 20 PM
Screenshot 2023-04-11 at 10 52 46 PM

File: script.js [hunk 2]
```diff
@@ -13,6 +13,189 @@ xMenu.addEventListener('click', () => {
...
+}</ul>
+      <hr>
+      <div class="popup_btn">
+        <button type="button" href="${
+  projects[cardNo].live_link
+}">See Live <i class="material-icons">arrow_forward</i></button>
+        <button type="button" href="${
+  projects[cardNo].source_link
+}">See source<i class="fa fa-github"></i></button>
+      </div>
```
Comment (huseyinbiyik) [inline script.js]: Please follow the design guideline on Figma and ensure that icons on the See Live and See Source are identical 👍  You can export as SVG from Figma.
Screenshot 2023-04-11 at 10 45 26 PM
Screenshot 2023-04-11 at 10 45 37 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Popups generated dynamically 👍 
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- PR description is descriptive 📝

## Required Changes ♻️
_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @mohametalmeari , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- All requested changes implemented :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**